libafl_libfuzzer/libafl_libfuzzer_runtime/src/options.rs

```diff
@@ -242,6 +242,9 @@ struct LibfuzzerOptionsBuilder<'a> {
     ignore_crashes: bool,
     ignore_timeouts: bool,
     ignore_ooms: bool,
+    no_ignore_crashes: bool,
```

Comment:
I think we should instead use an `enum` here, maybe something something like
`rust
enum Setting {
    Enabled,
    Disabled,
    Default,
}
```
or even have a `Tui` enum variant here?
Alternatively, maybe an `Option<bool>` would work?

Reply:
Yup, I think using `Option<bool>` is a much cleaner approach.